### PR TITLE
core: fix message about show status state

### PR DESCRIPTION
**Pull request by keszybz**
We would say "Enabling" also for SHOW_STATUS_AUTO, which is actually
"soft off". So just print the exact state to make things easier to understand.

For #14814.

### CODE REVIEW

File: src/core/manager.c [hunk 1]
```diff
@@ -4083,12 +4083,14 @@ void manager_set_show_status(Manager *m, ShowStatus mode) {
         if (!MANAGER_IS_SYSTEM(m))
                 return;
 
+        bool enabled = IN_SET(m->show_status, SHOW_STATUS_TEMPORARY, SHOW_STATUS_YES);
```
Comment (yuwata) [inline src/core/manager.c]: I think this should be `mode` instead of `m->show_status`.

Comment (keszybz) [reply src/core/manager.c]: Yep.

**Comment by keszybz** [comment]
Now this is a bit longer: a new mode show-status=error is added, which only shows errors. "quiet" implies show-status=error instead of show-status=auto as before. The logging of status switches is significantly improved.

@kchibisov could you check if this gives you the expected behaviour?

**Comment by kchibisov** [comment]
>could you check if this gives you the expected behaviour?

It doesn't give me expected behavior.

Log from booting with `quiet systemd.log-level=debug`.

[quite-debug.txt](https://github.com/systemd/systemd/files/4270718/quite-debug.txt)

**Comment by kchibisov** [comment]
Sorry I've attached a wrong log at first. It does improve the situation though. Now I see 3 messages during boot.

```
Welcome to Gentoo/Linux dracut-049(initramfs)!

[FAILED]Failed to start Dispatch Password Requests for Console Directory Watch.

Welcome to Gentoo/Linux!
```

I can't find this failed thing anywhere btw.

[quite-debug.txt](https://github.com/systemd/systemd/files/4270752/quite-debug.txt)

**Comment by keszybz** [comment]
```
Welcome to Gentoo/Linux dracut-049(initramfs)!
Welcome to Gentoo/Linux!
```
Duh, I forgot to update one place in the patch. This should be fixed now.

```
[FAILED]Failed to start Dispatch Password Requests for Console Directory Watch.
```
see this part in the log:
```
Feb 29 21:39:34 Kagami systemd[1]: /lib/systemd/system/systemd-ask-password-console.service:12: Executable "systemd-tty-ask-password-agent" not found in path "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
Feb 29 21:39:34 Kagami systemd[1]: systemd-ask-password-console.service: Unit configuration has fatal error, unit will not be started.
Feb 29 21:39:34 Kagami systemd[1]: systemd-ask-password-console.service: Failed to load configuration: Exec format error
```
It is fixed in https://github.com/dracutdevs/dracut/commit/16abea2592d7fee6166660098b647356d5a4cc14. 

OK, so I think this is all clear now: you had a failing unit, and this caused the logging to be turned on. I'll mark the last patch as fixing this issue, because it does make it trivial to see what error actually was instead of turning messages on.

**Comment by kchibisov** [comment]
>Duh, I forgot to update one place in the patch. This should be fixed now.

It's fixed now indeed. With you patch and with a patched dracut it seems to work as before. Thanks.

One subjective note which I'm not sure about, I feel like I've seen a state where systemd was shutting down a task during poweroff, but there wasn't an indicator for it like it was before (just blinking cursor for ~90 seconds). As I said I'm not sure that it actually true and I can't repro it on purpose, so just leaving this comment to double check things.

**Comment by yuwata** [comment]
LGTM.

**Comment by keszybz** [comment]
> One subjective note which I'm not sure about, I feel like I've seen a state where systemd was shutting down a task during poweroff, but there wasn't an indicator for it like it was before (just blinking cursor for ~90 seconds). As I said I'm not sure that it actually true and I can't repro it on purpose, so just leaving this comment to double check things.

The increase in the timeout that is applied to start (5s→25s) with "quiet" also applies to shutdown. But I think this is appropriate, since the logic is similar in both cases.

**Comment by keszybz** [comment]
bionic-amd64 and bionic-i386 fail with timeouts, doesn't seem to be related.

**Pull request merged**